events/event/migrations/0001_initial.py

```diff
@@ -1,8 +1,10 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.10.2 on 2016-10-20 07:18
+# Generated by Django 1.10.2 on 2016-11-09 07:43
```

Comment:
it is not ok to rewrite existing migration. You should add new for saving migrations history